sdk/samples/iot/hub/README.md

```diff
@@ -128,6 +128,24 @@ On failure, a status of `404` will be returned with an empty JSON payload.
 ### [IoT Hub C2D (Certificates)][c2d_sample]
 Receive and view incoming C2D messages using the IoT Hub Client.
 
+### [IoT Hub PnP (Certificates)][pnp_sample]
+Connect a PnP enabled device with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json).
+Note that due to limitations in the JSON builder, the values which are listed as doubles in the DTMI
+are instead `int32_t` here. This shortcoming is being tracked [here](https://github.com/Azure/azure-sdk-for-c/issues/881). In short, the capabilities are listed here:
+- **Methods**: Invoke a method called `getMaxMinReport` with JSON payload value `"since"` with an [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) value for start time for the report. The method sends a response containing the following JSON payload:  
+```json
+{
+  "maxTemp": 20,
+  "minTemp": 20,
+  "avgTemp": 20,
+  "startTime": "<ISO8601 time>",
+  "endTime": "<ISO8601 time>"
+}
+```
+with correct values substituted for each field.
+- **Telemetry**: Device sends a JSON message with the field name `temperature` and the `double` value of the temperature.
+- **Twin**: Desired property with the field name `targetTemperature` and the `double` value for the desired temperature. Reported property with the field name `maxTempSinceLastReboot` and the `double` value for the highest temperature.Note that part of the PnP spec is a response to a desired property update from the service. The device will send back a reported property with a similarly named property and a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description.
```

Comment:
I am having a bit of trouble parsing this paragraph since it reads like sentence fragments.

Nit: add space after period `... highest temperature. Note that part of the ...`
```suggestion
- **Twin**: Desired property with the field name `targetTemperature` and the `double` value for the desired temperature. Reported property with the field name `maxTempSinceLastReboot` and the `double` value for the highest temperature. Note that part of the PnP spec is a response to a desired property update from the service. The device will send back a reported property with a similarly named property and a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description.
```